source/slang/slang-emit-cpp.cpp

```diff
@@ -2059,15 +1999,69 @@ void CPPSourceEmitter::emitSimpleTypeImpl(IRType* inType)
     m_writer->emit(slice);
 }
 
-void CPPSourceEmitter::emitTypeImpl(IRType* type, const StringSliceLoc* nameLoc)
+void CPPSourceEmitter::_emitType(IRType* type, DeclaratorInfo* declarator)
```

Comment:
I'm confused why this can't just share its logic with the `_emitType` in the base `CLikeSourceEmitter`?